docs/sources/getting-started/_index.md

```diff
@@ -1,13 +1,11 @@
 +++
 aliases = ["/docs/grafana/latest/getting-started/", "/docs/grafana/latest/guides/what-is-grafana/"]
 title = "Getting started"
-weight = 10
+weight = 25
 +++
 
 # Getting started
```

Comment:
Should we change this to "Get started" given the discussion about not using gerunds?

Reply:
Good call, made that change.